intellij/src/saros/intellij/editor/EditorAPI.java

```diff
@@ -95,13 +101,68 @@ public static LineRange getLocalViewPortRange(
    * @return the current selection in the given editor
    */
   @NotNull
-  static TextSelection getSelection(@NotNull Editor editor) {
+  static Pair<TextSelection, Boolean> getSelection(@NotNull Editor editor) {
```

Comment:
As already written in a previous PR: I would expect that isBackwardsSelection could be moved in TextSelection.